### PR TITLE
Update the comments about oak-unsafe in oak_functions

### PR DESCRIPTION
**Pull request by rbehjati**

### CODE REVIEW

**Comment by jul-sh** [comment]
Thanks! Let's also update the relevant parts in `runner/src/internal.rs`

**Comment by rbehjati** [comment]
> Thanks! Let's also update the relevant parts in `runner/src/internal.rs`

Thanks. Done.

File: oak_functions/loader/Cargo.toml [hunk 1]
```diff
@@ -6,8 +6,8 @@ edition = "2018"
 license = "Apache-2.0"
 
 [features]
-# Feature that allows logging of potentially sensitive content.
-# Should only be used for debugging purposes.
+# Feature that allows logging of potentially sensitive content, or using experimental features.
+# Should only be used for debugging purposes or enabling experimental features.
```
Comment (jul-sh) [inline oak_functions/loader/Cargo.toml:10]: imo we can probs remove at that line at this point :)

**Review by jul-sh — APPROVED** [review]

File: oak_functions/loader/Cargo.toml (hunk 1 above)
Comment (rbehjati) [reply oak_functions/loader/Cargo.toml:10]: Do you mean that the entire comment about this feature can be removed?

**Pull request merged**